packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -1251,3 +1252,9 @@ function IsInScope(context: TCGCContext, entity: Operation): boolean {
   }
   return true;
 }
+
+export const $emptyStringAsNull: EmptyStringAsNullDecorator = (
+  context: DecoratorContext,
+  target: Model,
+  scope?: LanguageScopes,
+) => {};
```

Comment:
I think we need some validation in this decorator - for instance, if we insist it to apply to a model, we need to validate the model has a property whose type is string.
And if we change the target to property, we should validate it should only apply to a string-typed property.

Reply:
validation added.